src/services/dumpers/templates/forest-express/models/mongo-model.hbs

```diff
@@ -6,7 +6,7 @@ module.exports = (mongoose, Mongoose) => {
   // Learn more here: https://docs.forestadmin.com/documentation/reference-guide/models/enrich-your-models#declaring-a-new-field-in-a-model
   const schema = Mongoose.Schema({
   {{#each fields as |field|}}
-    {{wq field.name}}: {{#if field.ref}}{ type: {{field.type}}, ref: '{{field.ref}}' }{{else if (isObject field.type)}}{{>renderNested type=field.type level=2}}{{else}}{{field.type}}{{/if}},
+    {{wsc field.name}}: {{#if field.ref}}{ type: {{field.type}}, ref: '{{field.ref}}' }{{else if (isObject field.type)}}{{>renderNested type=field.type level=2}}{{else}}{{field.type}}{{/if}},
```

Comment:
This needs to be taken out from this PR in a fix PR. 

This prevents attributes without spaces or dashes to be quoted. Today, any of the attributes are being quoted, which is ugly.